experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -108,6 +108,25 @@ class Flowables {
     return Flowable<T>::create(std::move(lambda));
   }
 
+  template <typename T, typename TGenerator>
+  static Reference<Flowable<T>> fromGenerator(TGenerator generator) {
+    auto lambda = [generator = std::move(generator)]
+    (Subscriber<T>& subscriber, int64_t requested) {
+      int64_t generated = 0;
+      try {
+        while (generated < requested) {
+          subscriber.onNext(generator());
```

Comment:
@alexmalyshev Should I just emit one value at a time and let Flowable go thru the lambda multiple times? Flowables::range has the same problem. We can generate values even after cancel.
Technically speaking that is ok for the reactive streams protocol. It says after cancel we may receive some more values...